lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -329,41 +330,51 @@ def update_from_swarm_info(self, swarm_info):
             self.name = spec['Name']
 
         if self.labels is None:
-            self.labels = spec.get('Labels') or {}
```

Comment:
Is it safe to remove this as it was safe harbor for unpredictable value?

Reply:
Actually, that's a good point. I think I'll change this back, *and* add a test which makes sure that labels are kept.